src/app/views/Zap.svelte

```diff
@@ -17,60 +15,53 @@
   export let splits
   export let id = null
   export let anonymous = false
-  export let amount = getSetting("default_zap")
+  export let amount = getSetting("default_zap") as string
```

Comment:
Assertion is incorrect, it should be a number right?

Reply:
yes